### PR TITLE
Make updates to Deploy pricing calculator

### PR DESCRIPTION
**Pull request by thomasthesecond**
https://aptible.atlassian.net/browse/WWW-2

### CODE REVIEW

File: src/components/deploy/SupportPriceTable.js [hunk 1]
```diff
@@ -290,7 +282,7 @@ export default () => (
     <SupportRow />
     <Divider className={styles.beyondDivider} />
     <BeyondRow />
-    <Footnotes />
+    <Footnotes footnotes={footnotes} />
```
Comment (zkuzmic) [inline src/components/deploy/SupportPriceTable.js:285]: I'm seeing three footnote markers but only one footnote "Support via Slack..." etc.

File: src/components/pricing/DeployCalculator.js [hunk 2]
```diff
@@ -103,179 +162,247 @@ class DeployCalculator extends React.Component {
...
-                  <div className={styles.managedHidsCheckbox}>
-                    {this.state.managedHids &&
-                      <img src={checkmarkIcon} alt="checkmark" />
-                    }
-                  </div>
-                  With <Link to="/deploy/hids/">Managed HIDS</Link>
+                {/* <Link to="/deploy/hids/">Managed HIDS</Link> */}
+                {/* <div className={styles.lineItemAmount}>
+                  $0.02/GB/Hour
+                </div> */}
```
Comment (zkuzmic) [inline src/components/pricing/DeployCalculator.js:300]: Still needed?

**Comment by thomasthesecond** [comment]
Estimated backup costs isn't ready to go live yet. Next month sometime, learned that today myself.

File: src/components/pricing/DeployCalculator.js (hunk 2 above)
Comment (thomasthesecond) [reply src/components/pricing/DeployCalculator.js:300]: Nope, thanks

File: src/components/deploy/SupportPriceTable.js (hunk 1 above)
Comment (thomasthesecond) [reply src/components/deploy/SupportPriceTable.js:285]: 2 and 3 are described here

image

**Comment by thomasthesecond** [comment]
Agree about the hover, will fix

File: src/components/deploy/SupportPriceTable.js (hunk 1 above)
Comment (zkuzmic) [reply src/components/deploy/SupportPriceTable.js:285]: Ah, I see. I interpreted those as also referencing a footnote that I was expecting to be at the bottom of the page.

Comment (thomasthesecond) [reply src/components/deploy/SupportPriceTable.js:285]: Yeah good point, might be something to revisit in the future.

Comment (zkuzmic) [reply src/components/deploy/SupportPriceTable.js:285]: Yeah, we can go ahead with this for now

**Pull request merged**